api/modules/config.py

```diff
@@ -20,6 +20,51 @@ def define_string_list(name: str, args: bool=False, usage: str="") -> None:
       usage: When arg parsing fails, what to print for this setting's description.
     """
 
+def define_string(name: str, args: bool=False, usage: str="") -> None:
+    """
+    Defines a config setting of type `str`.
+
+    Allows the user invoking Tilt to configure a key named ``name`` to be in the
+    dict returned by :meth:`parse`.
+
+    To set a flag of this type named, e.g., `foo`, at runtime to "bar", run ``tilt up -- --foo bar``.
```

Comment:
Reorder this clause for clarity
```suggestion
    For instance, at runtime, to set a flag of this type named `foo` to value "bar", run ``tilt up -- --foo bar``.
```